Makefile.frag

```diff
@@ -66,3 +66,25 @@ libmongoc-version-current:
 
 libmongocrypt-version-current:
 	cd src/libmongocrypt/ && python etc/calc_release_version.py > ../LIBMONGOCRYPT_VERSION_CURRENT
+
+generate-function-map: all
```

Comment:
I didn't think of this before, but I think this warrants a new section in CONTRIBUTING.md.

Also, should this be a [phony target](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) since the name is not an actual filename? I think we may be missing that for a few other things as well, like `libmongoc-version-current` and `libmongocrypt-version-current`.

Reply:
Added paragraph in contribution docs and updated the list of phone targets.